hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-router/src/main/java/org/apache/hadoop/yarn/server/router/rmadmin/FederationRMAdminInterceptor.java

```diff
@@ -879,9 +879,11 @@ private DeregisterSubClusters deregisterSubCluster(String reqSubClusterId) {
       SubClusterState subClusterState = subClusterInfo.getState();
       long lastHeartBeat = subClusterInfo.getLastHeartBeat();
       Date lastHeartBeatDate = new Date(lastHeartBeat);
-
+      String heartBeatTimeOut =
```

Comment:
Don't we use somewhere else other than the log message?

Reply:
I will modify the code.